.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,7 +9,11 @@ Please include
 - links to issues that this PR addresses
 -->
 
+## Related issues/PRs
 
+<!-- Please include a reference to issues this PR fixes by adding `fixes #<issue_number>`. GitHub will then automatically close the issue once this PR is merged.
+If this PR is documentation for a feature, please add a reference to those PRs.
+-->
```

Comment:
Can we add a suggested format for these here so we get consistent results?

e.g.,

```
<!-- If appropriate, include a link to the issue this fixes -->
fixes <ISSUE LINK>

<!-- If appropriate, add links to any number of PRs documented by this PR -->
documents <PR LINK>
```

Reply:
Good call, I like your suggested format.